app/views/RoomView/index.js

```diff
@@ -131,6 +130,8 @@ export default class RoomView extends LoggedView {
 			} else {
 				EventEmitter.addEventListener('connected', this.handleConnected);
 			}
+
+			navigation.addListener('willBlur', () => this.hideActionSheet());
```

Comment:
I think this is not needed as RoomView will be removed from the stack, but test it.

If it's necessary, check if it's opened and move it to `componentWillUnmount`.
This is similar to `editCancel` and `replyCancel`.

Reply:
unfortunately the bottomsheet doesn't have function `onPositionChange` or something similar, but I did manage to do backdrop using `Animated.Code`

Reply:
It's necessary, because when I open Bottom Action Sheet in RoomView and then go to Threads Single View (thread chat) then ActionSheet is still displayed there and same happens when changing RoomViews(chats)

Reply:
And also I tried to move it to `componentWillUnmount` but as we use StackNavigator - the view in not unmounted when opening Threads View or Settings view and bottom sheet stays constantly opened.